src/CMakeLists.txt

```diff
@@ -106,3 +111,35 @@ set_target_properties(box2d PROPERTIES
 
 source_group(TREE "${CMAKE_CURRENT_SOURCE_DIR}" PREFIX "src" FILES ${BOX2D_SOURCE_FILES})
 source_group(TREE "${CMAKE_CURRENT_SOURCE_DIR}/../include" PREFIX "include" FILES ${BOX2D_HEADER_FILES})
+
+install(
+  TARGETS box2d
+  EXPORT box2dTargets
+  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+)
+
+install(
+  EXPORT box2dTargets
```

Comment:
Why don't you export it directly to box2dConfig? In this case I don't think an intermediate box2dTargets is needed.

Reply:
It's necessary for `box2dConfigVersion.cmake` below. And it's future-proof.

Reply:
CMake is automatically looking for that files when you call `find_package`. You don't need to include it manually. In fact, looking at the generated file I'm pretty sure this has no effect (not useful variable in there).